assets/css/main.scss

```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
   from { background-color: #ffb3c9}
   to {background-color: transparent}
 }
+
+/* ==== MAGICAL MODAL ==== */
+.modal {
+  align-items: center;
+  bottom: 0;
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 20;
+}
+
+.modal.is-active {
+  display: flex;
+}
+
+.modal-background {
+  background-color: rgba(10, 10, 10, .86);
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
```

Comment:
I think for position: absolute you only need to specify 
- either top or bottom
- either left or right
As far as I can remember, there's no need to specify, bottom and top and left and right.

Reply:
I just tried it... if I drop _any_ of left/right/top/bottom the background disappears.